cw-orch/src/daemon/sender.rs

```diff
@@ -16,10 +18,8 @@ use cosmwasm_std::Addr;
 use secp256k1::{All, Context, Secp256k1, Signing};
 use std::{convert::TryFrom, env, rc::Rc, str::FromStr};
 
-use tonic::transport::Channel;
-
 const GAS_LIMIT: u64 = 1_000_000;
-const GAS_BUFFER: f64 = 1.2;
+const GAS_BUFFER: f64 = 1.5;
```

Comment:
This gas buffer is annoying, any ideas on how we could improve on it?

Reply:
I don't think it's possible no. However, we could track transactions for failure due to gas cost and retry them with a higher gas_buffer if they fail because of that. 1.5 is a looot !